moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_trajectory.h

```diff
@@ -129,6 +132,11 @@ class ChompTrajectory
    */
   void fillInCubicInterpolation();
 
+  /**
+   * \brief receives the path obtained from OMPL and puts it into the appropriate trajectory format required for CHOMP
```

Comment:
Is MotionPlanDetailedResponse really OMPL-specific or can the trajectory be produced by other planners? If it's not OMPL-specific, then rename this to `initializeWithTrajectory` or something like that.

Reply:
actually STOMP can also be made to work with `MotionPlanDetailedResponse` as in [here](https://github.com/raghavendersahdev/moveit/blob/stomp_planner/moveit_planners/stomp/src/stomp_planner.cpp#L170) . I can possibly change it to `initializeWithTrajectory`

Reply:
done ! replaced function name with fillInFromTrajectory()